packages/stream_video_push_notification/lib/src/stream_video_push_notification.dart

```diff
@@ -174,13 +178,24 @@ class StreamVideoPushNotificationManager implements PushNotificationManager {
       return;
     }
 
-    void registerDevice(String token, bool isVoIP) {
-      _client.createDevice(
+    void registerDevice(String token, bool isVoIP) async {
+      final tokenKey = isVoIP ? 'userDeviceTokenVoIP' : 'userDeviceToken';
```

Comment:
Can we change this key to something unique and identifies this as a stream key? I feel like these are names that someone could use by accident in their own app.

Reply:
good point